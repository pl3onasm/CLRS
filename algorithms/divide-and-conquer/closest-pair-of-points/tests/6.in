250
(56.703482,186.511180),(43.967937,64.610296),(239.172178,142.169110),(14.814807,201.198691),(148.506613,220.934579),(20.082316,18.628197),(58.221316,172.451476),(57.662378,113.008128),(156.999032,207.013839),(129.125045,85.627484),(126.630972,95.408393),(182.379974,1.710935),(21.338563,12.574277),(170.629709,113.711738),(52.531084,213.781756),(207.625126,109.234567),(150.292936,1.593063),(173.844863,139.465114),(143.762174,188.659670),(90.663804,42.268786),(159.594248,110.746121),(60.896983,217.815564),(33.197597,118.559361),(80.823692,190.196629),(75.573200,209.948736),(25.824113,202.204172),(55.357129,208.204086),(203.915106,76.695692),(220.778364,124.544816),(190.407431,23.309448),(88.326572,148.032557),(132.544014,238.619507),(149.625621,56.388877),(128.084621,43.387794),(245.048547,218.748425),(85.656580,154.642795),(79.494546,146.553563),(122.458359,112.692143),(15.112924,203.282050),(52.888772,90.686124),(163.230787,78.712884),(42.890295,218.587916),(36.916971,246.805401),(45.283608,7.695334),(121.350217,235.691039),(31.004782,209.676788),(133.723596,163.548796),(198.296296,33.349216),(219.937674,76.380916),(76.737010,214.986220),(45.129342,162.393590),(119.629015,124.623888),(58.947153,242.087374),(237.316031,74.060077),(195.369424,40.204802),(164.746200,108.600210),(118.917687,207.636495),(77.188126,155.834657),(204.441897,122.471734),(163.529991,75.792113),(108.162773,194.534773),(35.468902,241.886369),(108.083569,233.765197),(25.235585,78.021243),(60.146114,101.972595),(43.007463,105.275456),(14.366185,162.636478),(229.899344,73.313338),(154.723851,217.215374),(147.373415,100.093275),(7.420177,62.119615),(208.693485,126.337863),(19.756111,35.881611),(32.172520,224.198007),(158.353345,195.702511),(49.990121,16.516118),(140.237285,85.459023),(8.402487,248.320854),(69.224220,33.638072),(76.342097,129.370334),(135.610666,119.349560),(234.645789,149.976851),(31.986038,214.545133),(223.290189,186.709889),(181.760507,120.663604),(36.803164,189.180684),(182.783220,245.496650),(65.518547,202.539330),(31.378261,97.691067),(176.737338,189.731606),(43.393578,226.727458),(206.247725,183.630863),(62.186481,214.650211),(181.951717,131.410701),(248.288283,8.293814),(10.781035,133.898949),(127.643374,245.426824),(33.875800,159.629412),(209.971957,7.165990),(96.339301,141.732464),(127.829594,133.142466),(80.913148,60.612813),(128.639115,146.431695),(13.152144,160.017376),(244.122762,189.889481),(99.748982,37.516340),(166.616939,55.996707),(221.147203,228.803420),(20.646918,153.098920),(110.214121,18.935201),(161.392734,120.995156),(152.834150,39.036108),(116.421981,186.709950),(198.665520,76.393938),(193.875940,45.004822),(218.126402,71.705534),(178.147287,49.039550),(132.318347,56.786403),(195.471245,145.470491),(216.803779,189.594006),(85.359972,66.552761),(227.110347,1.976912),(122.549468,198.257549),(230.780332,143.196386),(101.356469,90.994453),(162.131587,12.749203),(211.989610,64.965736),(51.785311,78.411590),(1.675687,0.450831),(154.805528,195.551627),(45.455653,122.931930),(17.257161,223.602940),(171.971480,149.575508),(30.389342,117.442725),(45.045999,247.193121),(57.036731,130.405971),(63.745882,34.147078),(132.382883,186.295349),(232.404627,113.163215),(79.491735,83.761097),(204.157668,241.623322),(96.510300,166.147278),(56.589058,148.295611),(244.558868,58.264744),(148.746442,149.364396),(3.816371,194.202095),(22.296326,21.073532),(167.805034,194.267806),(170.649040,198.194377),(61.710531,215.695039),(195.387498,118.747263),(96.101010,9.133379),(152.894341,228.483893),(195.428729,135.298968),(91.647107,24.920464),(219.060064,45.804775),(16.543785,65.570364),(211.952053,73.132843),(213.865975,206.510921),(131.397587,112.612416),(105.875316,135.213958),(56.814511,128.171643),(156.287490,224.619545),(72.439449,76.936529),(172.813922,134.149980),(42.631568,118.201419),(2.897243,138.732578),(127.334799,155.791584),(117.216470,72.763527),(41.090551,208.863577),(97.683991,10.150616),(4.668352,114.227776),(75.720980,216.620405),(187.360619,39.586954),(173.131325,68.758206),(152.199370,29.006642),(203.972164,209.013881),(157.178284,110.259654),(183.633426,229.617733),(187.196183,106.447348),(113.767714,229.827751),(224.648767,116.664957),(118.560329,101.983566),(22.456540,235.776799),(174.747093,63.547092),(194.640376,22.431084),(73.697707,199.308728),(136.658860,149.418687),(165.929133,74.019478),(189.005641,89.060458),(142.777684,91.205011),(118.067100,96.749848),(50.218892,25.245384),(207.009501,233.852319),(4.863117,144.205684),(90.299667,118.630831),(124.033435,64.948434),(235.295788,242.593764),(166.932000,7.752328),(228.370563,91.679093),(71.299420,173.010938),(114.110176,144.997127),(122.319666,0.769036),(44.415814,38.248798),(74.788514,233.421455),(127.309256,217.566197),(74.626466,245.376356),(64.316045,124.845359),(20.621740,21.325546),(108.697677,25.484857),(165.531230,198.997344),(144.115688,39.564666),(13.945777,129.411475),(32.158430,180.877777),(137.163804,10.528992),(22.556870,208.463223),(183.539930,136.667046),(103.460350,55.859596),(137.436082,147.876165),(94.108395,212.224596),(131.297620,221.417651),(179.790793,205.924086),(216.794007,244.106838),(80.769445,237.415747),(15.432384,189.467122),(12.900603,180.963614),(138.464466,157.016291),(220.528280,152.410243),(36.427767,2.686710),(83.288020,173.591570),(13.215702,105.844890),(132.054793,196.755632),(242.511936,235.515144),(2.615228,129.948018),(133.391309,96.723623),(92.172613,14.688928),(68.141274,21.963406),(220.613014,34.935281),(16.070244,51.382459),(22.351028,31.502628),(240.849581,35.251631),(212.466242,129.314047),(192.267922,182.994522),(31.724290,228.695689),(185.681231,115.012310),(152.287259,198.896933),(220.857200,34.342052),(145.652565,213.369136),(19.857196,148.267794),(93.317154,153.248505)
