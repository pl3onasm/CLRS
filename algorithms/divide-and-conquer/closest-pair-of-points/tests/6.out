Closest distance: 0.641520
Found between (220.613014,34.935281) and (220.857200,34.342052)
